protocol/chainlib/rest.go

```diff
@@ -440,6 +441,11 @@ func (rcp *RestChainProxy) SendNodeMsg(ctx context.Context, ch chan interface{},
 		defer res.Body.Close()
 	}
 
+	if res.StatusCode == 504 || res.StatusCode == 429 {
```

Comment:
can you export this to the errorHandler struct? so we can change it in the future in one place for all occurrences?